### PR TITLE
all terrain vehicle 

### PR DESCRIPTION
**Pull request by TroupeMaster313**
## Describe your project in 1-3 sentences
HELLO!,
I'm building a all terrain vehicle to test it on different kind of terrains and climates. It has been my dream to build an RC all terrain vehicle.

## Hack Club Slack handle

@troupemaster 

## Checklist before submitting proposal:

- [- ] I am a current high school student (or younger).
- [- ] I have submitted an RSVP at <https://hackclub.com/winter>
- [- ] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [- ] I have self-reviewed my project proposal.

### CODE REVIEW

File: Troupmaster313.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+| 3d printing filament                  |already have                                                                     | none  |
+| dremel tool                           |already have                                                                     | none  |  
+| screw drivers                         |already have                                                                     | none  | 
+| 3d printer                            |already have                                                                     | none  | 
+| soldering iron                        |already have                                                                     | none  | 
+| wire clipper                          |already have                                                                     | none  | 
+| nuts and bolts kit                    |https://rcprinter.com/products/nuts-and-bolts-kits?variant=42702968717473        | 74.99 | 6.99
+| charging cable for lipo battery       |https://www.amazon.com/Pairs-Female-Connector-Silicon-Battery/dp/B07QH249CR      | 8.99  | 5.99
```
Comment (bellesea) [inline Troupmaster313.md:48]:
```suggestion
| charging cable for lipo battery       |https://www.amazon.com/Pairs-Female-Connector-Silicon-Battery/dp/B07QH249CR/ref=asc_df_B07QH249CR/?tag=hyprod20&linkCode=df0&hvadid=361104274063&hvpos      | 8.99  | 5.99
```

File: Troupmaster313.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |/ref=asc_df_B07QH249CR/?tag=hyprod20&linkCode=df0&hvadid=361104274063&hvpos      |-------|
```
Comment (bellesea) [inline Troupmaster313.md:49]:
```suggestion
```

File: Troupmaster313.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |(link shortened for convinience)                                                 |-------|
```
Comment (bellesea) [inline Troupmaster313.md:50]:
```suggestion
```

File: Troupmaster313.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+| lipo battery x2                       |https://www.amazon.com/2200mAh-Helicopter-Airplane-Quadcopter-4-13x1-34x0-9      |26.60  | 5.99
```
Comment (bellesea) [inline Troupmaster313.md:51]:
```suggestion
| lipo battery x2                       |https://www.amazon.com/2200mAh-Helicopter-Airplane-Quadcopter-4-13x1-34x0-94inch/dp/B071R2SCWR/ref=asc_df_B071R2SCWR/tag=hyprod20&linkCode=df0&hvadid=198068753909&hvpos=&hvnetw=g&hvrand=17296433276288669141&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9052182&hvtargid=pla-358411390062&th=1      |26.60  | 5.99
```

File: Troupmaster313.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |4inch/dp/B071R2SCWR/ref=asc_df_B071R2SCWR/tag=hyprod20&linkCode=df0&hvadid=      |-------|
```
Comment (bellesea) [inline Troupmaster313.md:52]:
```suggestion
```

File: Troupmaster313.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |198068753909&hvpos=&hvnetw=g&hvrand=17296433276288669141&hvpone=&hvptwo=&hv      |-------|
```
Comment (bellesea) [inline Troupmaster313.md:53]:
```suggestion
```

File: Troupmaster313.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |qmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9052182&hvtargid=pla-358411390062      |-------|
```
Comment (bellesea) [inline Troupmaster313.md:54]:
```suggestion
```

File: Troupmaster313.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |&th=1                                                                            |-------|
+|11 teeth gear x2                       |https://www.ebay.com/itm/393380634608?chn=ps&mkevt=1&mkcid=28&srsltid=AeTunc     |11.82  |FREE
+|                                       |pfsl70oEdRACaARvLpMMBPEt2ruZx6WSJ_TEz_lPEM-jFgBEgmwnk                            |-------|
```
Comment (bellesea) [inline Troupmaster313.md:57]:
```suggestion
```

File: Troupmaster313.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |                                                                                 |-------|
+|brushless motors x2                    |https://www.walmart.com/ip/540-Brushed-Motor-RC-Crawler-Motor-35T-3-Slot-and     |56.06  |10.99
```
Comment (bellesea) [inline Troupmaster313.md:59]:
```suggestion
|brushless motors x2                    |https://www.walmart.com/ip/540-Brushed-Motor-RC-Crawler-Motor-35T-3-Slot-and-60A-ESC-RC-Car-ESC-Brushed-Electric-Speed-Controller-2A-BEC-for-1-10-RC-Car/722888429?wmlspartner=wlpa&selectedSellerId=101044639         |56.06  |10.99
```

File: Troupmaster313.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |-60A-ESC-RC-Car-ESC-Brushed-Electric-Speed-Controller-2A-BEC-for-1-10-RC-Car     |-------|
```
Comment (bellesea) [inline Troupmaster313.md:60]:
```suggestion
```

File: Troupmaster313.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |/722888429?wmlspartner=wlpa&selectedSellerId=101044639                           |-------|
```
Comment (bellesea) [inline Troupmaster313.md:61]:
```suggestion
```

File: Troupmaster313.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |                                                                                 |-------|
```
Comment (bellesea) [inline Troupmaster313.md:62]:
```suggestion
```

File: Troupmaster313.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+|screw sleeve                           |https://www.walmart.com/ip/JSSH-3-175mm-To-5mm-Pinion-Reducer-Sleeve-Adapter     |7.79   |free
```
Comment (bellesea) [inline Troupmaster313.md:63]:
```suggestion
|screw sleeve                           |https://www.walmart.com/ip/JSSH-3-175mm-To-5mm-Pinion-Reducer-Sleeve-Adapter-Shaft-For-3650-550-540-RC-Motor/494155400?wmlspartner=wlpa&selectedSellerId=101044662       |7.79   |free
```

File: Troupmaster313.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |-Shaft-For-3650-550-540-RC-Motor/494155400?wmlspartner=wlpa&selectedSellerId     |-------|
```
Comment (bellesea) [inline Troupmaster313.md:64]:
```suggestion
```

File: Troupmaster313.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |=101044662                                                                       |-------|
```
Comment (bellesea) [inline Troupmaster313.md:65]:
```suggestion
```

File: Troupmaster313.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |                                                                                 |-------|
```
Comment (bellesea) [inline Troupmaster313.md:66]:
```suggestion
```

File: Troupmaster313.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+|controller with reciver                |https://m.aliexpress.us/item/3256804682897437.html?_randl_currency=USD&src=g     |13.17  |FREE
```
Comment (bellesea) [inline Troupmaster313.md:67]:
```suggestion
|controller with reciver                |https://m.aliexpress.us/item/3256804682897437.html?_randl_currency=USD&src=google&aff_fcid=3f09faabbf65408d9cc01084209b5dff-1671575844607-09648-       |13.17  |FREE
```

File: Troupmaster313.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |oogle&aff_fcid=3f09faabbf65408d9cc01084209b5dff-1671575844607-09648-             |-------|
```
Comment (bellesea) [inline Troupmaster313.md:68]:
```suggestion
```

File: Troupmaster313.md [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+|                                       |                                                                                 |-------|
+|screw sleeve                           |https://www.walmart.com/ip/JSSH-3-175mm-To-5mm-Pinion-Reducer-Sleeve-Adapter     |7.79   |free
+|                                       |-Shaft-For-3650-550-540-RC-Motor/494155400?wmlspartner=wlpa&selectedSellerId     |-------|
+|                                       |=101044662                                                                       |-------|
+|                                       |                                                                                 |-------|
+|controller with reciver                |https://m.aliexpress.us/item/3256804682897437.html?_randl_currency=USD&src=g     |13.17  |FREE
+|                                       |oogle&aff_fcid=3f09faabbf65408d9cc01084209b5dff-1671575844607-09648-             |-------|       
+|                                       |(link shortened for convinience)                                                 |-------|
```
Comment (bellesea) [inline Troupmaster313.md:69]:
```suggestion
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @troupmaster313 thank you so much for sharing this!!! it's soooo cool :) 

to check do you have access to all the tools you mentioned? also, is there a model that you're modelling your vehicle after?

thank you!

ps. i added some formatting suggestions hehe, keeping the links together helps people click on it and view :)

File: Troupmaster313.md [hunk 20]
```diff
@@ -0,0 +1,78 @@
+# Project Name
+3d printed all terrain vehicle test
```
[the remark this answers is not in the document]
Comment (TroupeMaster313) [reply Troupmaster313.md:2]: Hello! I do have all the tools metioned in my request. There is also a model that i am basing my vehicle off of. I am planning on making the model more efficient by changing the shapes of the caterpillars and trying to decrease the pressure put onto the vehicles wheels.

[the remark this answers is not in the document]
Comment (bellesea) [reply Troupmaster313.md]: > Hello! I do have all the tools metioned in my request. There is also a model that i am basing my vehicle off of. I am planning on making the model more efficient by changing the shapes of the caterpillars and trying to decrease the pressure put onto the vehicles wheels.

awesome!! thank you hehe :)

**Review by bellesea — APPROVED** [review]
@TroupeMaster313 can't wait to see this come to fruition! best of luck and a super cool member of @hackclub/bank-ops will reach out w/ next steps soon :)

**Review by daisybanks — APPROVED** [review]
@TroupeMaster313 this looks really fun and cool! I'm so excited to see this built out in our 10 days of making! Our next step would be to hop on a quick onboarding call where I can explain how you can receive the hardware grant on Hack Club Bank. I'm going move communications over to Slack to make them a bit faster/easier. :)

**Pull request merged**